packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -112,116 +110,101 @@ const ProfilePanel: FC = () => {
                 timeout: 3000,
             });
         }
-    };
+    });
 
     return (
-        <div
-            style={{
-                height: 'fit-content',
-                display: 'flex',
-                flexDirection: 'column',
-            }}
-        >
-            <FormGroup
-                label="First name"
-                labelFor="first-name-input"
-                labelInfo="(required)"
-            >
-                <InputGroup
+        <form onSubmit={onSubmit}>
+            <Stack mt="md">
```

Comment:
only applied Stack to the inputs because of the messaging below the email input @IrakliJani 
![image](https://user-images.githubusercontent.com/7611706/235870454-ae25e3fe-8336-4180-a68c-2aa612b3cebf.png)